src/api/directinput/JoystickInterfaceDirectInput.h

```diff
@@ -43,7 +43,7 @@ namespace JOYSTICK
   private:
     bool InitializeDirectInput(void);
 
-    static BOOL CALLBACK EnumJoysticksCallback(const DIDEVICEINSTANCE *pdidInstance, VOID *pContext);
+    static BOOL EnumJoysticksCallback(const DIDEVICEINSTANCE *pdidInstance, VOID *pContext);
```

Comment:
I didn't move that method in JoystickInterface, only the similar method in JoystickDirectInput.
This one uses member variables so didn't want to touch it for this fix.